config/v3_2_experimental/schema/ignition.json

```diff
@@ -514,6 +514,9 @@
             },
             "shell": {
               "type": ["string", "null"]
+            },
+            "nonUnique": {
```

Comment:
The name `nonUnique` follows the existing pattern of naming fields after `useradd` options.  But I'm wondering if the name is too unclear in this case, since it doesn't mention UIDs.